client/lib/products-values/index.js

```diff
@@ -234,6 +234,12 @@ export function isJetpackScan( product ) {
 	return isJetpackScanSlug( product.product_slug );
 }
 
+export function isJetpackCloudProductSlug( productSlug ) {
```

Comment:
Not a blocker, but it seems like the function should be named a bit differently, something like `isJetpackCloudEligibleSlug`. Again, not a blocker.